addon/index.js

```diff
@@ -346,9 +346,13 @@ function updateExperiments() {
 function syncAllAddonInstallations(serverInstalled) {
   const availableIDs = Object.keys(store.availableExperiments);
   const clientIDs = Object.keys(store.installedAddons);
-  const serverIDs = serverInstalled
-    .filter(item => item.client_id === store.clientUUID)
-    .map(item => item.addon_id);
+  const serverIDs = [];
```

Comment:
Not really sure, I need to dig into DRF and see what that entails. Could be simple, but need to spend some hours figuring that out. That's why I suggest doing a tweak on the frontend for now to convert to the format until that's worked out.